src/dlg_ptt.cpp

```diff
@@ -116,24 +116,10 @@ ComPortsDlg::ComPortsDlg(wxWindow* parent, wxWindowID id, const wxString& title,
     wxStaticBoxSizer* staticBoxSizer17 = new wxStaticBoxSizer( serialSettingsBox, wxVERTICAL);
     mainSizer->Add(staticBoxSizer17, 1, wxEXPAND, 5);
     
-    wxStaticBox* pttBox = new wxStaticBox(serialSettingsBox, wxID_ANY, _("PTT Port"));
+    wxStaticBox* pttBox = new wxStaticBox(serialSettingsBox, wxID_ANY, _("PTT Out"));
```

Comment:
Can we pls use the nomenclature for PTT/PTT In, rather than moving to a PTT In/PTT Out?  I don't want the end users have to think about this step (am I configuring PTT In or Out?), as it will lead to confusion.

I'd like to emphasis that the PTT in feature is very much a low demand, one off feature, and should have minimal impact on the current PTT feature which is very important.  I want to make sure the tail doesn't wag the dog.

Reply:
Reverted back to "PTT Port".